### PR TITLE
[BLOG] Add welcome-tania-allard-as-labs-codirector

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-welcome-tania-allard-as-labs-codirector-quansight.vercel.app/blog/welcome-tania-allard-as-labs-codirector).

## Links to other Quansight blogs:
- Internal links:

## Text styling
- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/welcome-tania-allard-as-labs-codirector.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+with a new plugin architecture.
+
+![Logos of Data APIs, Ibis and PyData Sparse.](/posts/welcome-tania-allard-as-labs-codirector/dataapis_ibis_sparse.png)
+
+Our largest innovation initiative is the
+[Consortium for Python Data API Standards](https://data-apis.org/), which we
+created in mid-2020. This consortium brings together many array, tensor and
+dataframe library maintainers and has the ambitious aim to tackle the problem
+of fragmentation of Python array and dataframe libraries. We published our
+first RFC for an array API standard in November, and there's more to come in
```
Comment (MarsBarLee) [inline apps/labs/posts/welcome-tania-allard-as-labs-codirector.md:72]:
```suggestion
first RFC for an array API standard in November, and there's more to come in 2021.
```

File: apps/labs/posts/welcome-tania-allard-as-labs-codirector.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+2021.
```
Comment (MarsBarLee) [inline apps/labs/posts/welcome-tania-allard-as-labs-codirector.md:73]:
```suggestion

```

File: apps/labs/posts/welcome-tania-allard-as-labs-codirector.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+category: [Community]
+featuredImage:
+  src: /posts/welcome-tania-allard-as-labs-codirector/blog_feature_var1.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/welcome-tania-allard-as-labs-codirector/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+![Tania Allard.](/posts/welcome-tania-allard-as-labs-codirector/tania_allard.jpg)
```
Comment (MarsBarLee) [inline apps/labs/posts/welcome-tania-allard-as-labs-codirector.md:15]: In the original blog website, this image of Tania was the header image. Currently, I have the image in the top of the blog body.

This image's resolution is too small to be the feature or hero image. Should I get a higher resolution version of the same image to make it the hero and feature image?

Comment (noatamir) [reply apps/labs/posts/welcome-tania-allard-as-labs-codirector.md:15]: I found [this image](https://drive.google.com/file/d/1anr6tJ4UiyZO-p8oYZ5gDrnXOB64E0m6/view) on gdrive. @trallard should we use this, or do you have a larger version of the original?

**Comment by noatamir** [comment]
Looks good overall. Just waiting for the photo update.

File: apps/labs/posts/welcome-tania-allard-as-labs-codirector.md (hunk 3 above)
Comment (pavithraes) [reply apps/labs/posts/welcome-tania-allard-as-labs-codirector.md:15]: @trallard Pinging just surface this thread for you. :)

**Review by trallard — APPROVED** [review]

**Pull request merged**